shared/js/background/classes/score.es6.js

```diff
@@ -106,26 +120,69 @@ class Score {
                 beforeIndex += tosdrClassMap[this.tosdr.class]
                 afterIndex += tosdrClassMap[this.tosdr.class]
 
+                this.addDecision({
+                    change: tosdrClassMap[this.tosdr.class],
+                    index: beforeIndex,
+                    why: `Has tosdr class ${this.tosdr.class}`,
+                    tosdr: this.tosdr
+                })
+
             } else if (this.tosdr.score) {
                 let tosdrScore =  Math.sign(this.tosdr.score)
                 beforeIndex += tosdrScore
                 afterIndex += tosdrScore
+
+                this.addDecision({
+                    change: tosdrScore,
+                    index: beforeIndex,
+                    why: `Has tosdr score ${tosdrScore}`,
+                    tosdr: this.tosdr
+                })
+            } else {
+                this.addDecision({ change: 0, index: beforeIndex, why: `No tosdr` })
             }
         }
 
-        if (this.inMajorTrackingNetwork) beforeIndex++
+        if (this.inMajorTrackingNetwork) {
+            beforeIndex++
+            this.addDecision({ change: 1, index: beforeIndex, why: `In major tracking network` })
+        } else {
+            this.addDecision({ change: 1, index: beforeIndex, why: `Not in major tracking network` })
+        }
+
+        this.hasHTTPS = this.hasHTTPS || https.canUpgradeHost(this.domain)
```

Comment:
This is a temporary hack to guard against the HTTPS inconsistencies we were seeing. If we end up merging this into the develop branch, this is something we'll want to drop.

Reply:
The decision to do it this way could also be moved to a post processing step.

Reply:
That's a good idea, but that can potentially affect which grade letter is picked, and we'd need to copy over that logic to the postprocessing step. Not sure if we care that much about the grade letter, though.

Reply:
True. So from a testing standpoint I think this should be the default behavior, but then perhaps in the future an option to choose which method is used might be useful.